lockbox-ios/Presenter/ItemListPresenter.swift

```diff
@@ -13,11 +13,23 @@ protocol ItemListViewProtocol: class {
     func hideEmptyStateMessaging()
 }
 
+struct ItemsText {
```

Comment:
Nit: naming. 

Having that `s` in the middle will get old fast.

Suggest, perhaps: `NamedItemList`, or `TextAndItemList`.